docs/_static/version_switcher.json

```diff
@@ -0,0 +1,29 @@
+[
```

Comment:
this seems like a test right (based on the versions)?  would be nice to remove any in-process stuff before merge